src/main/scala/org/phenoscape/kb/queries/TaxaWithPhenotype.scala

```diff
@@ -180,21 +177,15 @@ object TaxaWithPhenotype {
       BlazegraphNamedSubquery(
         sparql"""
         SELECT DISTINCT ?phenotype WHERE {
-          ?p $rdfsSubClassOf/$relation/$rdfsSubClassOf/$PartOfSome $entity .
-          GRAPH $KBMainGraph {
-            ?phenotype $rdfsSubClassOf ?p .
-          }
+          ?phenotype $relation/$part_of $entity .
         }
         """
       )
     else
       BlazegraphNamedSubquery(
         sparql"""
         SELECT DISTINCT ?phenotype WHERE {
-          ?p $rdfsSubClassOf/$relation $entity .
-          GRAPH $KBMainGraph {
-            ?phenotype $rdfsSubClassOf ?p .
-          }
+          ?phenotype $rdfsSubClassOf/$relation $entity .
```

Comment:
I guess we don't need `$rdfsSubclassof` here too?

Reply:
We do in this case, otherwise it won't work for absence.